### ISSUE TITLE
large output gets truncated on xml reporter

### ISSUE THREAD

**Issue by pmlopes** [tav/nodelint#27]
I've a very large project and when i run the xml reporter i get truncated output, i hacked it by modifying the xml reporter to write to a file and that solves it for me, but i think it should be fixed, here is my modified xml.js:

```
// example usage:
//
//     $ nodelint path/to/file.js --reporter examples/reporters/xml.js
//
// this reporter produces jslint xml that can be automatically recognised by
// Hudson Violations Plugin http://wiki.hudson-ci.org/display/HUDSON/Violations+Plugin

var path = require('path'),
    fs = require('fs');

function reporter(results) {

  function escape(str) {
    return (str) ? str.replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/"/g, '&quot;') : '';
  }

  var i, error, len, file,
      dir = process.cwd(),
      xml = '<?xml version="1.0" encoding="UTF-8" ?>\n<jslint>\n';

  for (i = 0, len = results.length; i < len; i += 1) {
    file = path.join(dir, results[i].file);
    error = results[i].error;
    xml += '\t<file name="' + file + '">\n' +
           '\t\t<issue char="' + error.character + '" evidence="' + escape(error.evidence || '') +
           '" line="' + error.line + '" reason="' + escape(error.reason) + '"/>\n' +
           '\t</file>\n';
  }

  xml += '</jslint>';
  var outputFile = process.env.NODELINT_OUT;
  if(outputFile) {
    fs.writeFileSync(outputFile, xml + '\n');
  } else {
    console.log(xml);
  }
}
```

**Comment by Sannis** [comment]
How much lines/bytes is in your project nodelint output?

**Comment by pmlopes** [comment]
at the moment it is 44642 bytes (this is using my write to file quick fix), i also am using the latest nodelint from npm and node 0.4.11 on a ubuntu 10.04 LTS server, the same happens on Mac OS Lion on node 0.4.12

**Comment by pmlopes** [comment]
this issue does not happen anymore with node 0.6.2

**Comment by Sannis** [comment]
Thanks, Paulo.
